MantidQt/CustomInterfaces/src/Reflectometry/QtReflRunsTabView.cpp

```diff
@@ -46,13 +46,13 @@ void QtReflMainView::initLayout() {
   ui.splitterTables->setStretchFactor(0, 0);
   ui.splitterTables->setStretchFactor(1, 1);
 
+  // Create the DataProcessor presenter
   ReflGenericDataProcessorPresenterFactory presenterFactory;
```

Comment:
Not sure about this. I like the `PresenterFactory`, but the concrete type has effectively been hard-coded. I think it would be better to inject the Factory, probably into the constructor of the `QtReflRunsTabView`